components/Headers/PageHeader.js

```diff
@@ -177,6 +177,45 @@ const PageHeader = ({
                   </div>
                   <DarkModeToggle />
                 </div>
+                */}
+{/* Start- */}
+
+                <div className="hidden md:flex justify-end items-end w-auto md:flex-1 lg:w-0">
+              <form
+                onSubmit={handleSearch}
```

Comment:
This handleSearch is undefined for this page,
we need to define this function in same way as done [here](https://github.com/gita/bg-frontend/blob/b1981dac591fc633dd788d9a17f1cad323a26859/components/Headers/IndexHeader.js#L114)

Reply:
Just paste this in component and it should be good to go 

```jsx

  function handleSearch(e) {
    e.preventDefault();
   
    if(input?.trim().length <= 0) {
      return;
    }
    router.push(`/search?query=${input}`, undefined, { shallow: true });
  }

```